config/config.go

```diff
@@ -95,5 +97,9 @@ func ReadConfig(settings *Settings, filePath string) error {
 		ServiceMode = settings.Mode
 	}
 
+	if settings.SentryDSN == "" {
+		settings.SentryDSN = os.Getenv("SENTRY_DSN")
```

Comment:
SENTRY_DSN is used on our infrastructure to set up sentry DSN, it's always set on the unit the service is deployed